cfgov/jobmanager/models.py

```diff
@@ -102,8 +103,8 @@ def get_absolute_url(self):
 
     def save(self):
         if self.date_created == None:
```

Comment:
Yep, I agree there are some improvements that could be made to the existing job code. I tried to keep the scope of this PR limited to the particular issue of altering the table.